pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -124,6 +127,7 @@ func ProbeNetworkPlugins(confDir string, binDirs []string) []network.NetworkPlug
 		execer:         utilexec.New(),
 		confDir:        confDir,
 		binDirs:        binDirs,
+		cacheDir:       cacheDir,
```

Comment:
What does `cahceDir` do? Should it be configurable?

Reply:
It transfer to the libcni.
if the value is empty,the libcni will use a default cache dir to save the network result.